app/api/sessions.py

```diff
@@ -161,8 +161,9 @@ def after_update_object(self, session, data, view_kwargs):
                 frontend_url = get_settings()['frontend_url']
                 link = "{}/events/{}/sessions/{}" \
                     .format(frontend_url, event.identifier, session.id)
-                send_email_session_accept_reject(speaker.email, session, link)
-                send_notif_session_accept_reject(speaker, session.title, session.state, link, session.id)
+                if(not speaker.is_email_overridden):
```

Comment:
I guess you can remove the parenthesis here.

Reply:
Thanks for pointing out. Will update it